apple/OmnivoreKit/Sources/Views/ShareExtensionView.swift

```diff
@@ -188,12 +190,8 @@ public struct ShareExtensionChildView: View {
   }
 
   private func localImage(from: URL) -> Image? {
```

Comment:
from is kinda confusing as a local variable. maybe url instead

`func localImage(from url: URL) -> Image?`